src/pint/pintk/timedit.py

```diff
@@ -184,12 +184,11 @@ def applyChanges(self):
     def writeTim(self):
         filename = tkFileDialog.asksaveasfilename(title="Choose output tim file")
         try:
-            fout = open(filename, "w")
-            fout.write(self.editor.get("1.0", "end-1c"))
-            fout.close()
-            log.info("Saved timfile to %s" % filename)
-        except:
-            if filename == () or filename == "":
+            with open(filename, "w") as fout:
+                fout.write(self.editor.get("1.0", "end-1c"))
+            log.info(f"Saved timfile to {filename}")
+        except Exception:
+            if filename in [(), ""]:
                 log.warning("Write Tim cancelled.")
```

Comment:
Maybe this should be "Writing timfile cancelled"?

Reply:
Changed.